connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -213,4 +214,21 @@ object Connect {
     .version("3.5.0")
     .intConf
     .createWithDefault(200)
+
+  val CONNECT_ENRICH_ERROR_ENABLED =
```

Comment:
TBH I think we should get rid of this feature flag in SPARK-4.1.

Reply:
+1, for now, it may be better to have a flag to disable this feature in case something is not working.